### DISCUSSION TITLE
organisation submodules fork

### DISCUSSION THREAD

**Discussion by e-liouta** [upatras-hci/iv#47] [Q&A] [answered]
Πολύ καλησπέρα σας!

Δυσκολεύομαι να καταλάβω πως πρέπει να γίνει σωστά η προσθήκη των submodules στο αποθετήριο του οργανισμού. Δοκίμασα διάφορα αλλά το τελευταίο που έκανα είναι:
Έκανα fork το κεντρικό αποθετήριο site στον οργανισμό και έπειτα έκανα fork και κάθε submodule ξεχωριστά. Στη συνέχεια έκανα clone το αποθετήριο site του οργανισμού μας και αφού πραγματοποιήθηκε δοκίματα να ορίσω ως submodules τα άλλα αποθετήρια με την εντολή **git submodule add https://github.com/upatras-HCI-2022/_gallery.git** (αλλάζοντας κάθε φορά το link) αλλά παίρνω το error **fatal: '_gallery' already exists in the index** 
Μετά δοκίμασα να σβήσω τον φάκελο _gallery που προυπήρχε με το clone που έκανα αλλά ήταν κενός (χωρίς να ξέρω αν είναι σωστό) όμως και πάλι παίρνω το ίδιο error.
Θα μπορούσατε @epidrome να με κατευθύνετε ή οποιος άλλος έχει κάποια άλλη ιδέα από χθες που το συζητούσαμε.

**Comment by e-liouta** [comment]
Να συμπληρώσω ότι έκανα τα παραπάνω σκεπτόμενη να ακολουθήσω την ίδια διαδικασία για όλους τους φακέλους και μετά να κάνω push.

**Comment by epidrome** [comment, marked as answer]
@e-liouta φαίνεται σωστή η λογική σου, αλλά μάλλον θέλει εντολή git update γιατί υπάρχει ήδη αλλά δείχνει στα αρχικά, το σβήσιμο φακέλου θα μπορούσε να δουλέψει, αλλά τότε θέλει επεξεργασία με το χέρι σε κάποια αρχεία ρυθμίσεων

δείτε τους οδηγούς που έχουν ανεβάσει από το προπτυχιακό:
https://github.com/courses-ionio/help/discussions/1151

**Reply by e-liouta** [reply]
Ευχαριστώ πολύ! Δούλεψε!
